.changeset/poor-islands-battle.md

```diff
@@ -0,0 +1,10 @@
+---
+"@fluidframework/tree": minor
+---
+---
+"section": fix
```

Comment:
I think all of the SharedTree work falls under the "tree" section, even if it may also fall under another section.

Reply:
Interesting. Does it go under both sections here then? Or, should I remove "fix" and replace it with "tree"?

Reply:
You can remove "fix" and replace it with "tree"!

Reply:
Got it, thanks :)